tests/queries/0_stateless/01183_custom_separated_format_http.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+# Tags: no-tsan
+# FIXME should work with tsan
```

Comment:
If someone from the outside will read this comment, they can get the wrong impression that we have some code that does not pass TSan tests. But ClickHouse is 100% clean. Let's add a more descriptive comment.